docs/operation-guides/15-backup-volume-snapshots.md

```diff
@@ -143,3 +147,142 @@ spec:
                 # Delete old volume snapshots.
                 kubectl delete volumesnapshot -n {NAMESPACE} -l job=volume-snapshotter,name!=volume-snapshot-${RANDOM_ID}
 ```
+
+</p>
+</details>  
+
+
+
+## Create a volume snapshot for cloud providers
+
+The following instructions show how to create on-demand volume snapshots for cloud providers. Before you proceed, read the general instructions on [creating volume snapshots](/#tutorials-create-on-demand-volume-snapshots).
+
+<!-- why do I have to read those instructions first? -->
+
+<!-- it's completely unclear to me why Gardener providers are under a new headline. Why not put all five providers as tabs into the same div?? And then maybe even integrate this file with the other snapshot instruction -->
```

Comment:
I agree. :) GCP and AWS barely differ (this could be handled by some comment in the code sample or a note, while the Azure tab provides no steps after all, just a note you actually cannot do it (note would cover it). Also, the names Gardener GCP, Gardener Azure, etc, sound a bit akward.

Reply:
I agree they're awkward, I just didn't want to lose the fact we're talking about Gardener after I consolidated the tabs. 
Do you think I could remove "Gardener" from the headlines?

Reply:
I would leave Gardener and combine those 3 into one tab.

Reply:
:bulb: :bangbang: